jetty-server/src/main/config/modules/bytebufferpool.mod

```diff
@@ -0,0 +1,27 @@
+DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Configures the ByteBufferPool used by ServerConnectors.
+
+[xml]
+etc/jetty-bytebufferpool.xml
+
+[ini-template]
+### Server ByteBufferPool Configuration
+## Minimum capacity to pool ByteBuffers
+#jetty.byteBufferPool.minCapacity=0
+
+## Maximum capacity to pool ByteBuffers
+#jetty.byteBufferPool.maxCapacity=65536
+
+## Capacity factor
+#jetty.byteBufferPool.factor=1024
+
+## Maximum queue length for each bucket (-1 for unbounded)
+#jetty.byteBufferPool.maxQueueLength=-1
+
+## Maximum heap memory retainable by the pool (-1 for unlimited)
+#jetty.byteBufferPool.maxHeapMemory=-1
```

Comment:
I think that for jetty-10 we should consider setting a limit by default here.
Perhaps we need a -2 which sets the limit to a percentage of available memory?

Reply:
The percentage would be hard to guess. What about multiple pools? It may mess up with GC settings (e.g. people that have carefully tuned the generation sizes).
I don't know, I feel we should not be too smart. It's a new feature, let it cook for a while, and then if people uses it, we may change its default.